src/core/overloaded.h

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+//
+
```

Comment:
We have this but the implementation is slightly different because it adds an extra overloads. 
Also I am not sure if we can have the licence here ?

Reply:
Maybe you can also replace the overloaded in other files wit this include?